content/Timer_and_Divider_Registers.md

```diff
@@ -40,6 +40,8 @@ Bits 1-0 - Input Clock Select
 ```
 
 ::: tip NOTE
+Select the clock frequency first, then enable the Timer. Starting the Timer before
+or at the same time as the clock frequency is selected may result in excessive count up operation.
```

Comment:
That sounds copy-pasted from the official manual; is it? If so, then research as to what the actual behavior is may be needed. (Looking to eliminate that "may".)

Reply:
We could ask @Gekkio, if he shows up in a near future. In the meantime, following the recommendation doesn't hurt.

Reply:
There's [already a document detailing the circuits](https://gbdev.io/pandocs/#timer-obscure-behaviour), so we should be able to figure this out without bothering him ^^

Reply:
I agree. No point in reporting an apparently non-deterministic behaviour if we don't have enough proof